src/midi/midi.cpp

```diff
@@ -580,6 +713,16 @@ void init_midi_dosbox_settings(Section_prop& secprop)
 	const char* mputypes[] = {"intelligent", "uart", "none", 0};
 	str_prop->Set_values(mputypes);
 	str_prop->Set_help("MPU-401 mode to emulate ('intelligent' by default).");
+
+	auto* bool_prop = secprop.Add_bool("raw_midi_output", when_idle, false);
+	assert(bool_prop);
+	bool_prop->Set_help(
+	        "Enable raw, unaltered MIDI output (disabled by default).\n"
+	        "Many MIDI drivers used in games don't fully conform to the MIDI standard.\n"
+	        "Working with out-of-spec MIDI data recorded from such games in MIDI\n"
+	        "sequencers can result in various issues, e.g. hanging or missing notes.\n"
+	        "DOSBox tries to fix such problems by default; this should not normally affect\n"
+	        "the output in any audible way.");
```

Comment:
Yeah, and I'm not 100% happy with the wording either. Spent a long time on it... writing is hard 😅 (I can attest that by spending many dozens of hours on the doco project already...)

I think I want to keep the config name `raw_midi_output` though. Did not want to make it too specific and long (e.g., `explicit_note_offs_on_channel_mode_change_midi_messages` 😮‍💨, `compliant_midi_mode` -- compliant with what? 🤦🏻  ), and wanted to prevernt bizarre names too (e.g., `sanitise_midi_output`.

Reply:
Oh, yes - the conf name (raw_mini_output) is perfect and is very self-documenting as-is. It tells me that there is some checking and potential rectifying being done in the default scenario. 

So I think the description just has to basically confirm that, and to let users know that if midi doesn't sound right, they could try letting the raw steam out (because maybe some "defect" is actually part of the original experience, or some like that 😅!)